core/src/main/java/com/google/cloud/sql/CredentialFactory.java

```diff
@@ -17,11 +17,12 @@
 package com.google.cloud.sql;
 
 import com.google.api.client.auth.oauth2.Credential;
```

Comment:
Unused import?

Reply:
It's actually used in the comment but maybe the comment should be updated? I left it as is because I figured that in most cases it would be a `Credential` that is returned.

Reply:
Ah, yeah that's fine then.